src/app/(admin)/settings/components/sections/account.tsx

```
@@ -0,0 +1,27 @@
+export function AccountSettings() {
+    return (
+        <div>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)